pkg/crc/machine/hyperkit/constants.go

```diff
@@ -1,3 +1,5 @@
+//+build darwin build
```

Comment:
It seems using a `crc` build tag when building the `crc` binary (ie making this line `//+build darwin !crc`) would mean less changes to `Makefile`, I can post this alternate version.